### PR TITLE
Fix: step hike compact component

### PR DESCRIPTION
**Pull request by Isaiah-Hamilton**
## What kind of change does this PR introduce?

The step hike compact component has an issue where the last step has a line that leads to no next step.

## What is the current behavior?

CleanShot 2023-06-14 at 21 40 13@2x

## What is the new behavior?

CleanShot 2023-06-14 at 21 40 34@2x

### CODE REVIEW

**Comment by gregnr** [comment]
There's no way to use CSS to show/hide this eh? Something like `:last-child`

**Comment by Isaiah-Hamilton** [comment]
> There's no way to use CSS to show/hide this eh? Something like `:last-child`

Sadly no sense each step thinks that its the last-child. for example:

CleanShot 2023-06-14 at 22 14 20@2x

**Comment by gregnr** [comment]
Right, I was hoping we could use last child selector from the parent targeting the child, something like:

https://stackoverflow.com/questions/19569130/apply-css-last-child-to-an-element-that-is-not-the-immediate-child-in-ul-li

Not sure if possible with tailwind though.

**Comment by Isaiah-Hamilton** [comment]
> Right, I was hoping we could use last child selector from the parent targeting the child, something like:
> 
> https://stackoverflow.com/questions/19569130/apply-css-last-child-to-an-element-that-is-not-the-immediate-child-in-ul-li
> 
> Not sure if possible with tailwind though.

Oh I see what you mean. Update the code, 100% css!

File: apps/docs/components/StepHikeCompact/index.tsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ const StepHikeCompact: FC<IStepHikeCompact> & IStepHikeCompactSubcomponents = ({
 
 const Step: FC<IStep> = ({ children, title, step }) => {
   return (
-    <div className="relative pb-8">
+    <div className="relative pb-8 last-step">
```
Comment (Isaiah-Hamilton) [inline apps/docs/components/StepHikeCompact/index.tsx:32]: Couldn't come up with anything better than `last-step`.

Comment (MildTomato) [reply apps/docs/components/StepHikeCompact/index.tsx:32]: we can use tailwind for this.
it would be something like:

`last:[&div][&div]-bg-transparent`

Comment (MildTomato) [reply apps/docs/components/StepHikeCompact/index.tsx:32]: great fix though, looks much better!!

Comment (MildTomato) [reply apps/docs/components/StepHikeCompact/index.tsx:32]: sorry this is it

`last:[&_div]:[&_div]:bg-transparent`

**Review by MildTomato — DISMISSED** [review]

File: apps/docs/components/StepHikeCompact/index.tsx (hunk 1 above)
Comment (Isaiah-Hamilton) [reply apps/docs/components/StepHikeCompact/index.tsx:32]: `last:[&_div]:[&_div]:bg-transparent` doesn't work but if you rearrange the order to `[&_div]:[&_div]:last:bg-transparent` it does begin work. weird fix but finally working.

**Review by gregnr — APPROVED** [review]

**Pull request merged**